docs/dictionary.rst

```diff
@@ -0,0 +1,1101 @@
+
+see also :ref:`dict GCP`  
+
+Further reading: `<https://docs.aws.amazon.com/index.html?nc2=h_ql_doc_do>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict BD Catalyst:
+
+BD Catalyst
+-----------
+*abbreviation for* :ref:`dict BioData Catalyst`  
+
+
+.. updated 2022-05-16  
+
+
+
+.. _dict BDC:
+
+BDC
+---
+pronounced "bee-dee-see"  
+
+*abbreviation for* :ref:`dict Biodata Catalyst`  
+
+
+.. updated 2022-05-16  
+
+
+
+.. _dict Biodata Catalyst:
+
+Biodata Catalyst
+----------------
+	An initiative funded by :ref:`dict NHLBI` to connect several cloud-based bioinformatics platforms together to increase reproducibility in bioinformatics. Involves Dockstore, :ref:`dict Terra`, Seven Bridges, Gen3, and PIC-SURE.  
+
+Further reading: `<https://biodatacatalyst.nhlbi.nih.gov/>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict Cancer Genomics Cloud:
+
+Cancer Genomics Cloud
+---------------------
+	A cloud platform by :ref:`dict Seven Bridges` and funded by :ref:`dict NCI` for bioinformatics analysis.  
+
+
+.. updated 2022-05-16  
+
+
+
+.. _dict categories:
+
+categories
+----------
+	A group of workflows or tools on Dockstore with a similar scientific purpose.  
+
+
+.. updated 2022-05-16  
+
+
+
+.. _dict CGC:
+
+CGC
+---
+*abbreviation for* :ref:`dict Cancer Genomics Cloud`  
+
+
+.. updated 2022-05-16  
+
+
+
+.. _dict CLI:
+
+CLI
+---
+*abbreviation for* Command Line Interface  
+
+	A program that can be interacted with on the command line, usually via "Terminal" on MacOS and Linux or "cmd"/Command Prompt on Windows. CLI programs generally do not have a graphical user interface.  
+
+Further reading: `<https://en.wikipedia.org/wiki/Command-line_interface>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict cloud computing:
+
+cloud computing
+---------------
+	Doing computational tasks on a remote machine that is made available on-demand without the user having to manage all aspects of it. Generally implies that the user is essentially renting computational resources from someone else. Well-known cloud providers include :ref:`dict GCP`, :ref:`dict AWS`, Microsoft Azure, and Alibaba Cloud.  
+
+Further reading: `<https://en.wikipedia.org/wiki/Cloud_computing>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict collections:
+
+collections
+-----------
+	A group of workflows or tools on Dockstore associated with a particular :ref:`dict organization`.  
+
+
+.. updated 2022-05-16  
+
+
+
+.. _dict Common Workflow Language:
+
+Common Workflow Language
+------------------------
+	A workflow language that describes how to run command-line tools. CWL is based on Java and can use Java commands within its own commands. :ref:`dict WDL` and CWL are relatively similar in principle, and code written in one language can often be translated into the other with some workarounds, but they are two different standards and each have unique features.  
```

Comment:
CWL doesn't have much to do with Java, should probably just take that out. It can use CLI programs from slightly above.

Reply:
I suppose I was incorrect when I was talking about Java, but I think the ability to use Javascript all over the place is an important feature. Notably, there is no equivalent to this in WDL. 

https://www.commonwl.org/user_guide/13-expressions/index.html

Reply:
Oh, Javascript expressions. 
Yes, I think it could be worthwhile to point out Javascript expressions.